tests/plugins/font/fontcommand.js

```diff
@@ -0,0 +1,853 @@
+/* bender-ckeditor-plugins: font,toolbar,basicstyles,table,list */
```

Comment:
I'm wondering what to do with missing issue tags (`(#3728)`) here. Adding issue tags for every unit test seems weird as it's a lot of duplication but will pay off with better test discoverability. Maybe do you have some idea how we could handle this case?

Reply:
It definitely looks weird added to every single test :(

Maybe there could be added a global tag `/* bender-tags: 3728 */`. WDYT?

I also wonder how useful this tag could be here for general purpose unit tests? Maybe it's not necessary to add it at all?
The issue doesn't fix any specific edge case situation, which reference might be useful to understand reproduction steps. Do you have some idea how the issue reference here might be useful?

Reply:
Maybe let's just go with
```
/* bender-tags: 3728 */
```
? 
Even when there are more tests added to this test file, it still applies to the main ticket about this feature. WDYT?

Reply:
@msamsel could you refer to issue tags?